release/rllib_tests/learning_tests/run.py

```diff
@@ -3,12 +3,21 @@
 Runs Atari/PyBullet benchmarks for all major algorithms.
 """
 
+import argparse
 import json
 import os
 from pathlib import Path
 
 from ray.rllib.utils.test_utils import run_learning_tests_from_yaml
 
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--smoke-test",
+    action="store_true",
+    default=False,
+    help="Finish quickly for training.")
+args = parser.parse_args()
+
```

Comment:
Nit: Should this be in the __main__ block below?

Reply:
I copied it from one of the other (non-RLlib) scripts. :D 
In RLlib's example scripts, we also always leave it out of the main block. But yeah, probably shouldn't be outside of its in case you treat this as a module (which probably never happens :) ).